### PR TITLE
Improve hellothere flex cards accessibility

### PR DESCRIPTION
**Pull request by pmario**
This PR should improve the accessibility settings for screen readers. Tested with Narrator Windows-11

The PR

- Adds a new `alt-text` field to every image I could find. Especially those used for HelloThere - flex cards
- Improve flex-card procedure to render alt-text for images, so they can be read by screen readers
- It aria-hides the new-badge because it causes problems -- Full card text is read out 2 times

### CODE REVIEW

**Comment by Jermolene** [comment]
Great idea, thanks @pmario

**Comment by pmario** [comment]
For me it works OK, but I do not really need / use a screen reader. When I test keyboard handling, I switch it on from time to time, to see if I can reach every element in the ui. So I can hear, what narrator sees. 

I found out, that it wants to read the link URLs from the flex cards, which contain all the URL escape characters like eg: `%20` and so on. Which does not make real sense. So I tried to improve it. 

I'll create a post at Talk and ask screen reader users if the changes actually are an improvement. So we should get some feedback there.

File: editions/tw5.com/tiddlers/images/Funding.png.meta [hunk 1]
```diff
@@ -1,3 +1,4 @@
 title: Funding.png
 type: image/png
 tags: picture
+alt-text: Hand over a debit-card.
```
Comment (CrossEye) [inline editions/tw5.com/tiddlers/images/Funding.png.meta:4]: 😃

**Comment by CrossEye** [comment]
@pmario:
> I found out, that it wants to read the link URLs from the flex cards, which contain all the URL escape characters like eg: %20 and so on. 

Perhaps one day #7990 will fix this.

File: editions/tw5.com/tiddlers/images/Funding.png.meta (hunk 1 above)
Comment (pmario) [reply editions/tw5.com/tiddlers/images/Funding.png.meta:4]: You are right, that's not the best thing to do but IMO the image shows that. **Do you know a better description?**

**Comment by pmario** [comment]
@Jermolene -- There has not been much feedback at Talk - So I think it should be merged, then we will see.

File: editions/tw5.com/tiddlers/hellothere/HelloThere.tid [hunk 2]
```diff
@@ -1,6 +1,6 @@
 created: 20130822170200000
 list: [[A Gentle Guide to TiddlyWiki]] [[Discover TiddlyWiki]] [[Some of the things you can do with TiddlyWiki]] [[Ten reasons to switch to TiddlyWiki]] Examples [[What happened to the original TiddlyWiki?]]
-modified: 20240627165458407
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/hellothere/HelloThere.tid:3]: We normally only change the modified date of HelloThere when there is a new release

**Review by Jermolene — COMMENTED** [review]
Thanks @pmario I think you need to merge master back into this – see 8e8d5060e568d73fc96c90a3f89deb024f076a40 for changes to the Federatial logo

**Comment by Jermolene** [comment]
Hi @pmario there is a conflict that prevents merging

**Comment by pmario** [comment]
@Jermolene -- This is documentation only and conflicts have been resolved.

**Comment by Jermolene** [comment]
Thanks @pmario

**Pull request merged**